user-form/src/App.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useContext } from "react";
+import "./App.css";
+import "./Assets/scss/index.scss";
+import Nav from "./Components/NavBar";
+import { Route, Routes } from "react-router-dom";
+import Home from "./Components/Home";
+import UserList from "./Components/UserList";
+import jsonData from "../src/data.json";
+import Details from "./Components/Details";
+
+import Context from "./ContextApi/ContextForApp";
+
+function App() {
+  const [userData, setUserData] = useState(jsonData, "jsonData");
+
+  return (
+    <>
+      <Context>
+        <Nav />
+        <div className="container">
+          <Routes>
+            <Route
+              path="/"
```

Comment:
lets extract path in a variable and use in route.

const HOME={
INDEX: "/"
}


<Route path={HOME.INDEX} element={<Home userData={userData} setUserData={setUserData} />} />